### ISSUE TITLE
使用coco datasets读进loader，结果全是0数组

### ISSUE THREAD

**Issue by kkpssr** [PaddlePaddle/PaddleDetection#9092]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleDetection/issues)，没有发现相似的bug。I have searched the [issues](https://github.com/PaddlePaddle/PaddleDetection/issues) and found no similar bug report.

### Bug描述 Describe the Bug

使用coco datasets读进loader，loss一直是nan，打印data发现全是0数组
08/14 19:14:05] ppdet.engine INFO: Epoch: [0] [ 0/61] learning_rate: 0.024000 loss_vfl: inf loss_bbox: nan loss_dfl: nan loss: nan eta: 14:24:30 batch_cost: 2.8344 data_cost: 2.3220 ips: 8.4673 images/s
![image](https://github.com/user-attachments/assets/744a7396-f60a-4da6-a48a-6bb449b03f53)

### 复现环境 Environment

paddle 2.6

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [ ] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by UserWangZz** [comment]
你好，可以提供一个10张左右的测试数据集我们这边尝试复现一下吗？

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
